### PR TITLE
Draft Cleanup

### PR DESCRIPTION
**Pull request by lxgreen**
## REFACTORING

- [x] e2e should use  only RCE@Next + RCV@Next
- [x] e2e content snapshots should be RichContent
- [ ] content fixtures should be RichContent
- [x]  bundle-analyzer should use  only RCE@Next + RCV@Next
- [x] plugins: createDraftPlugin, createToolbar, insert-buttons, inline-buttons removed
- [x] ricos-editor: draft-related RicosEditor cleanup; EditorCommands cleanup
- [x] ricos-common: RicosEngine cleanup; move relevant types to ricos-types
- [x] editor: move AddPluginMenu to toolbars-ui
- [x] ricos-content: remove preview related code
- [x] plugin-commons: draft-related infra cleanup
- [x] editor-common: draft-related infra cleanup
- [x] common: draft-related infra cleanup
	- [x]  draft content validator service @lxgreen 
	- [x]  draft getContentSummary replace by richContent getContentSummary @talevy17 
	- [x]  draftUtils
	- [x]  innerRCEBlocksUtils
	- [x]  viewerUtils
- [x] link-preview: SettingsModal relies on editor-common/LinkModal which relies on draftUtils
- [x] unsupported blocks: move component to tiptap-editor & deprecate whole package @lxgreen 
- [x] orchestrator: schema entity cleanup (remove draftContent)
- [x] remove preview
- [x] remove ricos-viewer
- [x] remove viewer
- [x] remove editor
- [x] remove toolbars
- [x] remove toolbars-new
- [x] remove headers-markdown
- [x] remove plugin-map

## FINALIZATION

- [x] example app should use RCE@Next + RCV@Next
- [ ] storybook should use RCE@Next + RCV@Next
- [ ] docs cleanup (update deps, update docs to RCE/RCV@NEXT
- [ ] deprecate packages:
	- [ ] editor
	- [ ] viewer
	- [ ] ricos-viewer
	- [ ] preview
	- [ ] toolbars
	- [ ] toolbars-new
	- [ ] plugin-map
	- [ ] plugin-unsupported-blocks
	- [ ] plugin-headers-markdown
- [ ] [deprecate](https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions) packages on npm
- [x] ensure yarn.lock does not contain deprecated ricos packages
- [ ] update plugin template to tiptap plugin structure

### CODE REVIEW

File: e2e/cypress/support/commands.ts [hunk 1]
```diff
@@ -133,19 +117,20 @@ const COMMANDS = {
...
-          contentState = (win as any).__CONTENT_STATE__;
-        })
-        .then(() => {
-          const result = checkValidity(contentState, contentSchema);
-          if (!result.valid && result.errors) {
-            result.errors.forEach(error => console.error('schema validation error:', error)); // eslint-disable-line no-console
-          }
-          expect(result.valid).to.equal(true);
-        });
+      //TODO: add schema validation?
```
Comment (maimonav) [inline e2e/cypress/support/commands.ts:120]: remove

**Comment by maimonav** [comment]
@talevy17 can you please check the deploy-example action? it fails

**Comment by lxgreen** [comment]
@talevy17 if you add a table, the Viewer crashes the whole example app -- is this known issue?

**Comment by talevy17** [comment]
> @talevy17 if you add a table, the Viewer crashes the whole example app -- is this known issue?

wasn't aware of it, probably something new
checking. @lxgreen

**Comment by maimonav** [comment]
@danarish4 The embed test is failing, can you please check?

**Comment by maimonav** [comment]
@danarish4 @tamrat-bay @talevy17 I see that all tests passed (besides twitter toolbar which is skipped right?)
What's the status on the new/changed baselines?

**Review by talevy17 — COMMENTED** [review]
opening link modal crashes the editor, is it a known issue? @maimonav

**Comment by maimonav** [comment]
> opening link modal crashes the editor, is it a known issue? @maimonav
@talevy17 Hmmm no, it should work, and it worked before the last commit

**Pull request merged**